composer.json

```diff
@@ -31,7 +31,8 @@
         "white-october/pagerfanta-bundle"      : "^1.0"
     },
     "require-dev": {
-        "sensio/generator-bundle"              : "~3.0",
+        "phpunit/phpunit"                      : "^4.8 | ^5.0",
```

Comment:
the Composer recommendation is to use ``||`` for the OR operator rather than ``|``, because it is the operator used by other semver-based dependency managers